active_directory_ldap/help.md

```diff
@@ -19,20 +19,25 @@
 
 ## Setup
 
-To connect, you must have NTLM credentials.
-
-Please make sure you enter your credentials with the DOMAIN\username format.
```

Comment:
Should we add these back? icon-plugin automatically removes everything under Setup now - bug. I always do a `git add -p` to verify the changes I want to keep as a quikc workaround.

Reply:
ya we should

Reply:
going to add these to Requirements for now so they say around